websites/T/timolia/dist/metadata.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Timolia",
+    "author": {
+      "name": "Lennart™",
+      "id": "0001"
```

Comment:
Id should be your Discord ID, not your discriminator. 
Example (mine): 241278257335500811